docs/source/en/perf_infer_gpu_one.md

```diff
@@ -17,7 +17,17 @@ rendered properly in your Markdown viewer.
 
 In addition to this guide, relevant information can be found as well in [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
 
-## Better Transformer: PyTorch-native transformer fastpath
+## BetterTransformer
+
+BetterTransformer API converts 🤗 transformers models to make them use PyTorch-native transformer fastpath that calls optimized kernels such as Flash Attention under the hood.  
```

Comment:
You can copy from here https://github.com/huggingface/transformers/pull/25265/files#r1297372831

Reply:
Done!